src/test/java/com/gamsa/history/service/HitstoryServiceTest.java

```diff
@@ -17,21 +23,29 @@ public class HitstoryServiceTest {
             .actId(1L)
             .build();
 
+    ReviewService reviewService = new ReviewService(
```

Comment:
p4;  stub으로 생성하는 것 보다는 mock 객체를 사용하셔서 쓰는게 좋을 거 같아요

```suggestion
   @Mock
   ExistUserRepository mockExistUserRepository;
   
   @Mock
   QuestionRepository questionRepository;
   
  @InjectMock
  ReviewService reviewService;
```